python/features/steps/general_purpose_steps.py

```diff
@@ -107,6 +127,22 @@ def check_model_fail(context, model_file):
     context.output_message = output_message
 
 
+@when('"{command}" is called from terminal with the flag "{flags}"')
```

Comment:
@bunchw guess you didn't see the second comment. These should be used for all of the command executing when statements. Remember, testing should model usage. We don't inject spaces for not providing args or flags, so that shouldn't be what our test does. So, we should make steps that accommodate the four ways to utilize a command.